calculator.rb

```diff
@@ -0,0 +1,52 @@
+# Baseline
+
+# The program should ask the user for an operation (string or numeric symbol) and two numbers.
+
+# Primary Requirements
+
+# The program should use the input operation and two numbers to provide the result of applying the 
+# operation to the two numbers.
+
+# The program should have support for these four operations: addition, subtraction, multiplication, 
+# and division.
+# The program should accept both the name (add) and the symbol (+) for each possible operation.
+
+puts "give me a an operation, it can be a string or numeric symbol:"
+operation = gets.chomp
+
+puts "Please give me 2 numbers you'd like to do math on:"
+num1 = gets.chomp.to_i
+num2 = gets.chomp.to_i
+
+
+def add(num1, num2)
+	return num1 + num2
+end
```

Comment:
You're using tabs for indentation - please use two spaces per level of indentation instead. It doesn't make a big difference one way or the other, but keeping our style consistent across the class will make it much easier when we start doing group projects.

You can set your editor up to do this automatically, let me know if you need help with this.